### DISCUSSION TITLE
[item] shows material instead of item name

### DISCUSSION THREAD

**Discussion by daoshii** [Zrips/CMI#6393] [Feature Requests]
It would be nice if [item] in chat shows the items name instead of the material.

Pros:
- Many servers use custom items with custom names
- It wont be as ugly and chat would look nicer
- It would completely null the need to get InteractiveChat
- A player will no longer know what the vanilla material is of custom modeled items

Cons:
- none

![item](https://user-images.githubusercontent.com/61712934/143495347-ea6a4c79-5b85-4851-9a46-d92efa3371ff.png)

**Comment by daoshii** [comment]
NEVERMIND, I found where you can change it to %cmi_iteminhand_displayname%, thanks to gecko

**Comment by CloudeLecaw** [comment]
i also would like to see an option to see the displayed itemname on [item]
i know u see it when you hover over it.. but it would be nice not need to look/hover at every diamond sword offer if its the rare one that i wanted....
